File: ce/customer-engagement/on-premises/snippets/visualbasic/CRMV8/marketingvb/vb/distributecampaignfrommarketinglist10.vb
' <snippetdistributecampaignfrommarketinglist10>


	' Remove the marketing list from the campaign activity.
          Dim removeFromCampaignActivity =
              New RemoveItemCampaignActivityRequest With
              {
                  .CampaignActivityId = _campaignActivityId,
                  .ItemId = _copiedMarketingListId
              }

	_serviceProxy.Execute(removeFromCampaignActivity)
	Console.WriteLine("  Removed the marketing list from the campaign activity.")

' </snippetdistributecampaignfrommarketinglist10>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/marketing/cs/distributecampaignfrommarketinglist10.cs
// <snippetdistributecampaignfrommarketinglist10>


 // Remove the marketing list from the campaign activity.
 var removeFromCampaignActivity = new RemoveItemCampaignActivityRequest
 {
     CampaignActivityId = _campaignActivityId,
     ItemId = _copiedMarketingListId
 };

 _serviceProxy.Execute(removeFromCampaignActivity);
 Console.WriteLine("  Removed the marketing list from the campaign activity.");

// </snippetdistributecampaignfrommarketinglist10>